payments-app-extension-offsite/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,23 @@
+[[extensions]]
+name = "{{ name }}"
+type = "payments_app_extension"
+handle = "{{ handle }}"
+target = "offsite"
```

Comment:
Also, other extensions define the target inside a `targetting` object like:

```toml
  [[extensions.targeting]]
  target = "...."
```

Did you discuss this structure with anyone from app foundations?

Reply:
I posted a request to review this PR on their slack channel: https://shopify.slack.com/archives/C01D7U7AJ5C/p1704886259783989